pwiz_tools/Shared/PanoramaClient/PanoramaFormUtil.cs

```diff
@@ -0,0 +1,272 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Windows.Forms;
+using Newtonsoft.Json.Linq;
+
+namespace pwiz.PanoramaClient
+{
+    public enum ImageId
+    {
+        panorama,
+        labkey,
+        chrom_lib,
+        folder
+    }
+
+    internal class PanoramaFormUtil
+    {
+
+        public string Server;
```

Comment:
I don't think Server, User and Pass need to be instance variables.  They are used only in the `AddChildFiles` method but that method is duplicated here.  I think the one you are using is in `FilePicker`.